system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -688,6 +690,7 @@ class CLdapConnection : implements ILdapConnection, public CInterface
         }
     }
 
+private:
     virtual int connect(const char* ldapserver, const char* protocol)
```

Comment:
Is it possible this version of connect is being used? (is it possible a plugin which resides in a separate repo is currently referencing this method)? Most likely ok, but making sure we always consider when changing access rules.